assets/airlines/aal.json

```diff
@@ -1,67 +1,56 @@
-{
-    "icao": "aal",
-    "name": "American Airlines",
-    "callsign": {
-        "name": "American",
-        "callsignFormats": [
-            "####",
-            "###",
-            "##"
-        ]
-    },
-    "fleets": {
-        "default": [
-            ["A21N", 10],
-            ["A319", 133],
-            ["A320", 48],
-            ["A321", 219],
-            ["A332", 15],
-            ["A333", 9],
-            ["B38M", 24],
-            ["B738", 304],
-            ["B752", 34],
-            ["B763", 17],
-            ["B772", 47],
-            ["B77W", 20],
-            ["B788", 20],
-            ["B789", 22],
-            ["E190", 19]
-        ],
-        "long": [
-            ["A332", 15],
-            ["A333", 9],
-            ["B752", 34],
-            ["B763", 17],
-            ["B772", 47],
-            ["B77W", 20],
-            ["B788", 20],
-            ["B789", 22]
-        ],
-        "short": [
-            ["A21N", 10],
-            ["A319", 133],
-            ["A320", 48],
-            ["A321", 219],
-            ["B38M", 24],
-            ["B738", 304],
-            ["E190", 19]
-        ],
-        "tncmShort": [
-            ["B738", 304]
-        ],
-        "tncmLong": [
```

Comment:
As you have removed this fleet, you need to remove any references to this fleet in any airport files, in this case, TNCM.

Reply:
Looked through TNCM, there's no reference to the deleted fleet.